__init__.py

```diff
@@ -267,15 +267,27 @@ def on_websettings_changed(self):
             # mycroft-playback-control.mycroftai:PlayQueryTimeout
             self.refresh_saved_tracks()
 
-    def load_credentials(self):
-        """Retrieve credentials from the backend and connect to Spotify."""
+    def load_local_creds(self):
+        try:
+            creds = load_local_credentials(self.settings['user'])
+            spotify = SpotifyConnect(client_credentials_manager=creds)
+        except Exception:
+            self.log.exception('Couldn\'t fetch credentials')
+            spotify = None
+        return spotify
+
+    def load_remote_creds(self):
         try:
             creds = MycroftSpotifyCredentials(self.OAUTH_ID)
-            self.spotify = SpotifyConnect(client_credentials_manager=creds)
+            spotify = SpotifyConnect(client_credentials_manager=creds)
         except HTTPError:
             self.log.info('Couldn\'t fetch credentials')
-            self.spotify = None
+            spotify = None
+        return spotify
 
+    def load_credentials(self):
+        """Retrieve credentials from the backend and connect to Spotify."""
+        self.spotify = self.load_local_creds() or self.load_remote_creds()
```

Comment:
loading local first makes sense after running the script i think
(but if the user entered them in https://account.mycroft.ai/skills, it should fallback to that? or wait for propogation) should be fine :checkered_flag:

Reply:
load_remote_creds() is the old system using the mycroft-oauth credentials (still have the hope that spotify will come back to me with permission to use their api) So if the user has not set up local credentials it will try to use the remote ones.

Reply:
(maybe worth a comment or docstring)? up to you :)

Reply:
lgtm

Reply:
this was fixed :heavy_check_mark: